web/src/lib/components/user-settings-page/user-usage-statistic.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+  import {
+    getAlbumStatistics,
+    getAssetStatistics,
+    type AlbumStatisticsResponseDto,
+    type AssetStatsResponseDto,
+  } from '@immich/sdk';
+  import { onMount } from 'svelte';
+  import { t } from 'svelte-i18n';
+
+  let timelineStats: AssetStatsResponseDto = $state({
+    videos: 0,
+    images: 0,
+    total: 0,
+  });
+
+  let favoriteStats: AssetStatsResponseDto = $state({
+    videos: 0,
+    images: 0,
+    total: 0,
+  });
+
+  let archiveStats: AssetStatsResponseDto = $state({
+    videos: 0,
+    images: 0,
+    total: 0,
+  });
+
+  let trashStats: AssetStatsResponseDto = $state({
+    videos: 0,
+    images: 0,
+    total: 0,
+  });
+
+  let albumStats: AlbumStatisticsResponseDto = $state({
+    owned: 0,
+    shared: 0,
+    notShared: 0,
+  });
+
+  const getUsage = async () => {
+    [timelineStats, favoriteStats, archiveStats, trashStats, albumStats] = await Promise.all([
+      getAssetStatistics({ isArchived: false }),
+      getAssetStatistics({ isFavorite: true }),
+      getAssetStatistics({ isArchived: true }),
+      getAssetStatistics({ isTrashed: true }),
+      getAlbumStatistics(),
+    ]);
+  };
+
+  onMount(async () => {
+    await getUsage();
+  });
+</script>
+
+<section class="my-6">
+  <p class="text-xs dark:text-white uppercase">{$t('photo_and_video')}</p>
+  <table class="w-full text-left mt-4">
+    <thead
+      class="mb-4 flex h-12 w-full rounded-md border bg-gray-50 text-immich-primary dark:border-immich-dark-gray dark:bg-immich-dark-gray dark:text-immich-dark-primary"
+    >
+      <tr class="flex w-full place-items-center">
+        <th class="w-1/4 text-center text-sm font-medium">{$t('view')}</th>
+        <th class="w-1/4 text-center text-sm font-medium">{$t('photo')}</th>
+        <th class="w-1/4 text-center text-sm font-medium">{$t('video')}</th>
+        <th class="w-1/4 text-center text-sm font-medium">{$t('total')}</th>
+      </tr>
+    </thead>
+    <tbody class="block w-full overflow-y-auto rounded-md border dark:border-immich-dark-gray">
+      <tr
```

Comment:
This row is repeated four times, it would be a great use case for [snippets](https://svelte.dev/docs/svelte/snippet)